packages/babel-plugin-transform-typescript/test/fixtures/enum/non-scoped/output.js

```diff
@@ -3,7 +3,7 @@ var E = /*#__PURE__*/function (E) {
   E[E["y"] = 2] = "y";
   return E;
 }(E || {});
-E = /*#__PURE__*/function (E) {
+E = function (E) {
```

Comment:
This was not needed, terser doesn't remove used variables (E) anyway.

Reply:
I'm inclined to keep it, maybe in the future terser can be implemented or other compressors can be used?

Reply:
Done!